packages/cli/src/databases/utils/transformers.ts

```diff
@@ -2,8 +2,13 @@
 import { IPersonalizationSurveyAnswers } from '../../Interfaces';
 
 export const idStringifier = {
-	from: (value: number): string | number => (value ? value.toString() : value),
-	to: (value: string): number | string => (value ? Number(value) : value),
+	from: (value: number): string | number => (typeof value === 'number' ? value.toString() : value),
+	to: (value: string): number | string => (typeof value === 'string' ? Number(value) : value),
+};
+
+export const lowerCaser = {
+	from: (value: string): string => value,
+	to: (value: string): string => (typeof value === 'string' ? value.toLowerCase() : value),
```

Comment:
In `to` I would lowercase always. `lowercaser` applies only to `string` columns, so passing anything else to the corresponding object property will be already disallowed by TS/TypeORM.

Reply:
thought so as well, but typeorm thaught me differently. When we query for email it passes an object as the value for whatever reason, that's why we need this syntax